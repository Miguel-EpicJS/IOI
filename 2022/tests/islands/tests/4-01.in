4e5150f5a11b21d9ac3b44f2259e7a3452343ad4
100 400
74 54
74 54
35 26
35 26
21 77
21 77
38 48
38 48
83 91
83 91
7 51
7 51
49 0
49 0
5 39
5 39
83 14
83 14
88 51
88 51
83 39
83 39
29 66
29 66
11 41
11 41
3 13
3 13
51 23
51 23
3 49
3 49
70 54
70 54
37 91
37 91
92 2
92 2
77 97
77 97
41 40
41 40
23 0
23 0
63 97
63 97
58 51
58 51
38 37
38 37
84 27
84 27
31 83
31 83
96 73
96 73
54 9
54 9
68 5
68 5
58 51
58 51
44 1
44 1
28 59
28 59
40 51
40 51
3 25
3 25
70 49
70 49
43 55
43 55
28 20
28 20
92 34
92 34
24 33
24 33
89 47
89 47
92 4
92 4
89 97
89 97
48 1
48 1
62 72
62 72
98 12
98 12
89 50
89 50
34 75
34 75
81 72
81 72
89 61
89 61
5 6
5 6
89 52
89 52
89 28
89 28
44 60
44 60
33 67
33 67
81 55
81 55
26 46
26 46
86 12
86 12
33 13
33 13
43 6
43 6
79 59
79 59
31 48
31 48
29 47
29 47
48 90
48 90
28 21
28 21
43 74
43 74
89 84
89 84
58 93
58 93
28 12
28 12
38 2
38 2
57 9
57 9
75 31
75 31
81 42
81 42
36 51
36 51
41 51
41 51
90 67
90 67
91 51
91 51
15 70
15 70
45 27
45 27
49 0
49 0
89 26
89 26
70 91
70 91
24 68
24 68
49 21
49 21
17 51
17 51
82 44
82 44
33 68
33 68
3 60
3 60
56 16
56 16
63 45
63 45
30 52
30 52
69 35
69 35
35 51
35 51
81 14
81 14
4 65
4 65
19 12
19 12
83 62
83 62
17 23
17 23
63 67
63 67
25 20
25 20
81 6
81 6
93 37
93 37
61 59
61 59
82 67
82 67
11 0
11 0
24 44
24 44
29 80
29 80
82 55
82 55
70 90
70 90
7 58
7 58
79 36
79 36
29 95
29 95
62 22
62 22
68 9
68 9
76 32
76 32
4 16
4 16
88 51
88 51
7 97
7 97
35 98
35 98
32 87
32 87
44 6
44 6
81 72
81 72
23 7
23 7
28 90
28 90
78 79
78 79
86 75
86 75
21 88
21 88
10 85
10 85
78 98
78 98
38 33
38 33
36 18
36 18
67 27
67 27
82 60
82 60
47 65
47 65
71 55
71 55
22 0
22 0
39 2
39 2
38 2
38 2
0 91
0 91
82 64
82 64
81 94
81 94
80 52
80 52
44 5
44 5
24 66
24 66
22 11
22 11
85 17
85 17
37 91
37 91
24 80
24 80
87 57
87 57
23 97
23 97
97 20
97 20
68 53
68 53
82 80
82 80
85 6
85 6
88 94
88 94
86 9
86 9
89 71
89 71
95 12
95 12
37 15
37 15
28 39
28 39
28 72
28 72
22 25
22 25
9 65
9 65
57 92
57 92
39 53
39 53
95 27
95 27
34 91
34 91
90 55
90 55
63 39
63 39
10 51
10 51
92 51
92 51
40 67
40 67
47 27
47 27
51 20
51 20
1 77
1 77
8 76
8 76
63 60
63 60
38 73
38 73
19 69
19 69
68 71
68 71
63 94
63 94
32 97
32 97
78 91
78 91
3 48
3 48
7 91
7 91
46 10
46 10
47 2
47 2
14 71
14 71
53 64
53 64
18 19
18 19
81 30
81 30
44 73
44 73
38 60
38 60
76 84
76 84
43 15
43 15
88 8
88 8
40 78
40 78
70 0
70 0
35 52
35 52
86 16
86 16
