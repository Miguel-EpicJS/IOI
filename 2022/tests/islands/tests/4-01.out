91041670623a69783795ed82bda33426101ce311
OK
1
10
276 152 28 42 277 276 42 28 152 277
